draft-vvv-webtransport-quic.md

```diff
@@ -82,17 +82,23 @@ unidirectional streams, bidirectional streams and datagrams.
 
 # Introduction
 
+WebTransport [OVERVIEW] is a protocol framework that enables clients constrained
+by the Web security model to communicate with a remote server using a secure
+multiplexed transport.  This document describes QuicTransport, a transport
+protocol that uses a dedicated QUIC [QUIC] connection and provides support for
+unidirectional streams, bidirectional streams and datagrams.
+
 QUIC [QUIC] is a UDP-based multiplexed secure transport.  It is the underlying
 protocol for HTTP/3 {{?I-D.ietf-quic-http}}, and as such is reasonably expected
 to be available in web browsers and server-side web frameworks.  This makes it a
 compelling transport to base a WebTransport protocol on.
 
-This document defines QuicTransport, an adaptation of QUIC to WebTransport
-model.  The protocol is designed to be low-overhead on the server side, meaning
-that server software that already has a working QUIC implementation available
-would not require a large amount of code to implement QuicTransport.  Where
-possible, WebTransport concepts are mapped directly to the corresponding QUIC
-concepts.
+This document defines QuicTransport, an instantiation of WebTransport which is
```

Comment:
I actually have trouble understanding this sentence now.

Reply:
Fixed